### ISSUE TITLE
dumpconfig doesn't work

### ISSUE THREAD

**Issue by J-A-M-E-5** [ethereum/go-ethereum#26328]
#### System information

Geth version: Version: 1.11.0-unstable
OS & Version: Linux
Commit hash : 6891288787df7e4d342fef82d7c7581458f6b6b8

#### Expected behaviour
`$ geth --mainnet --datadir /node/geth dumpconfig`

_Config file dumped._

#### Actual behaviour
```
$ geth --mainnet --datadir /node/geth dumpconfig
INFO [12-08|04:21:15.611] Maximum peer count                       ETH=50 LES=0 total=50
INFO [12-08|04:21:15.615] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
INFO [12-08|04:21:15.622] Set global gas cap                       cap=50,000,000
toml: cannot marshal nil *types.Block
```

_Returns to terminal._

#### Steps to reproduce the behaviour
`$ geth --mainnet --datadir /node/geth dumpconfig`

**Comment by rjl493456442** [comment]
It's fixed in #26330, please try the master again.

**Comment by J-A-M-E-5** [comment]
> It's fixed in #26330, please try the master again.

Can confirm it is now working to my expectations. Thanks all for your quick fix. 🍺
